adoc/scripts/reflow.py

```diff
@@ -726,18 +535,27 @@ def reflowAllAdocFiles(folder_to_reflow, args):
     parser.add_argument('-out', action='store', dest='outDir',
                         default='out',
                         help='Set the output directory in which updated files are generated (default: out)')
+    parser.add_argument('-nowrite', action='store_true',
+                        help='Do not write output files, for use with -check')
+    parser.add_argument('-check', action='store', dest='check',
+                        help='Run markup checks and warn if WARN option is given, error exit if FAIL option is given')
+    parser.add_argument('-checkVUID', action='store', dest='checkVUID',
+                        help='Detect duplicated VUID numbers and warn if WARN option is given, error exit if FAIL option is given')
     parser.add_argument('-tagvu', action='store_true',
                         help='Tag un-tagged Valid Usage statements starting at the value wired into reflow.py')
     parser.add_argument('-nextvu', action='store', dest='nextvu', type=int,
                         default=None,
-                        help='Specify start VUID to use instead of the value wired into vuidCounts.py')
+                        help='Tag un-tagged Valid Usage statements starting at the specified base VUID instead of the value wired into reflow.py')
     parser.add_argument('-maxvu', action='store', dest='maxvu', type=int,
                         default=None,
                         help='Specify maximum VUID instead of the value wired into vuidCounts.py')
     parser.add_argument('-branch', action='store', dest='branch',
-                        help='Specify branch to assign VUIDs for.')
+                        help='Specify branch to assign VUIDs for')
     parser.add_argument('-noflow', action='store_true', dest='noflow',
-                        help='Do not reflow text. Other actions may apply.')
+                        help='Do not reflow text. Other actions may apply')
+    parser.add_argument('-margin', action='store', type=int, dest='margin',
+                        default='76',
+                        help='Width to reflow text, defaults to 76 characters')
```

Comment:
It looks like commit aa7f7a41187765c6603e8569a228df0c7da810f4 did not actual change the default margin width.  I thin these two lines above are what we need to change.

Reply:
Oh, good catch. I will just do a big grep on `76`.